examples/experimental/dagster-airlift/README.md

```diff
@@ -265,14 +272,15 @@ defs = build_defs_from_airflow_instance(
 
 For all other operator types, we recommend creating a new factory class whose arguments match the inputs to your Airflow operator. Then, you can use this factory to build definitions for each Airflow task.
 
-For example, our `load_raw_customers` task uses a custom `LoadCSVToDuckDB` operator. We'll define a `CSVToDuckDBDefs` factory to build corresponding software-defined assets:
+For example, our `load_raw_customers` task uses a custom `LoadCSVToDuckDB` operator. We'll define a function  `load_csv_to_duckdb_defs` factory to build corresponding software-defined assets:
```

Comment:
Another pass on this sentence would be nice. Feels like it throws forward a lot of verbiage in a slightly confusing way

Reply:
per offline discussion I was just doing a light pass here, rather than working on core content.